test/evergreen.yml

```diff
@@ -2765,6 +2765,13 @@ tasks:
             # By default the test_push_pull uses dir_store as a storage source.
             ./test_push_pull -PT
 
+  - name: dirty-upgrade-compatibility-test
```

Comment:
nit: seems like there is a convention here - starting with `compatibility-test-`

Reply:
Thanks! Fixed in 88253ff.